### ISSUE TITLE
[SBP - MS1] Bump versions

### ISSUE THREAD

**Issue by arturgontijo** [hashed-io/hashed-substrate#272]
**Description**

It's important to not leave your current deps versions so outdated in comparison to the Substrate releases.

**Suggested Solution**

Update from `v0.9.27` to `v0.9.33`.

**Comment by tlacloc** [comment]
Hello @arturgontijo!

I'm working on finishing this issue but I'll like some help clarifications regarding on how to import the dependencies for newest version of polkadot cumulus, I'm trying to run the parachain template in this repo importing all the dependencies but everytime I try to import it I got this error

```
the package `hashed-parachain-runtime` depends on `cumulus-pallet-aura-ext`, with features: `try-runtime` but `cumulus-pallet-aura-ext` does not have these features.


failed to select a version for `cumulus-pallet-aura-ext` which could resolve this conflict
```

I'm working on the bracn [272-sbp-ms1-bump-versions](https://github.com/hashed-io/hashed-substrate/tree/272-sbp-ms1-bump-versions)

**Comment by arturgontijo** [comment]
Hey @tlacloc let me check that for you.

**Comment by arturgontijo** [comment]
@tlacloc let me know if you can keep going with this one:

https://github.com/arturgontijo/hashed-substrate/tree/v0.9.33

I started it from the current `master`, btw.

**Comment by arturgontijo** [comment]
There will be some add/removal of types from imported pallets, but it should be straightforward to "fix" them.

Feel free to post any issue here =)

**Comment by tlacloc** [comment]
> can keep going with this o

Yep, I cloned it, compiled and It looks like it works! Awesome!

**Comment by tlacloc** [comment]
Closed as is now updated on #317
